docs/reference/contrib/searchpromotions.md

```diff
@@ -50,3 +52,58 @@ To retrieve a list of promoted search results for a particular search query, you
     {% endfor %}
 </ul>
 ```
+
+### Managing stored search queries
+
+The `searchpromotions` module keeps a log of search queries as well as the number of daily hits through the `Query` and `QueryDailyHits` models.
+
+#### `Query.get()`
+
+Retrieves a stored search query or creates a new one if it doesn't exist.
+
+#### `add_hit()`
+
+Records another daily hit for a search query by creating a new record or incrementing the number of hits for an existing record. Defaults to using the current date but an optional `date` parameter can be passed in.
+
+#### Example search view
```

Comment:
The original content in `wagtail.search`'s documentation should be removed, and the accompanying template should be put here as well.